src/components/NavBar.tsx

```diff
@@ -43,6 +44,10 @@ const NavBar = (user : any) => {
     const history = useHistory();
     const [userName, setUserName] = useState<string>('');
     const [userPicture, setUserPicture] = useState<string>('');
+    // const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+    const [open, setOpen] = useState(false);
```

Comment:
`open` is probably not a great name, better with something like `avatarDropdown` or `avatarMenuOpen`. Also its bad practice in ts to not specify types `useState<boolean>(false)`
```suggestion
    const [avatarMenuOpen, setAvatarMenuOpen] = useState<boolean>(false);
```